model/src/main/kotlin/config/AdvisorConfiguration.kt

```diff
@@ -31,7 +31,10 @@ import com.fasterxml.jackson.annotation.JsonTypeInfo
     JsonSubTypes.Type(NexusIqConfiguration::class),
     JsonSubTypes.Type(VulnerableCodeConfiguration::class)
 )
-sealed class AdvisorConfiguration
+data class AdvisorConfiguration(
```

Comment:
Please remove the above annotations; they should not be needed anymore if no sealed class is used.